packages/core/src/common/selection.ts

```diff
@@ -47,4 +50,14 @@ export namespace UriSelection {
         return [];
     }
 
+    export async function toClipboard(uris: URI[], clipboardService: ClipboardService, messageService: MessageService): Promise<void> {
```

Comment:
looks like `toClipboard` function has little in common with a selection,
maybe it would be better to move it at the place of usage?

Reply:
> maybe it would be better to move it at the place of usage?

I think, that the better place for this function will be `packages/core/src/browser/common-frontend-contribution.ts`, where command binds.

Reply:
Fixed